src/components/WalletAssets.vue

```diff
@@ -209,13 +210,35 @@ export default class WalletAssets extends Mixins(LoadingMixin, FormattedAmountMi
       }
     }
     &-value {
+      height: 16px;
```

Comment:
Could you use variable $basic-spacing-medium based on var(--s-basic-spacing) please?